internal/api/v1/routes/routes.go

```diff
@@ -24,22 +24,28 @@ var DefaultBaseURL = fmt.Sprintf("http://localhost:%s", DefaultPort)
 // Route names for lookup
 const (
 	// Jobs routes
-	CreateJob = "CreateJob"
-	GetJob    = "GetJobStatus"
-	ListJobs  = "ListJobs"
+	CreateJob    = "CreateJob"
+	GetJobStatus = "GetJobStatus"
+	ListJobs     = "ListJobs"
+	SearchJobs   = "SearchJobs"
+	UpdateJob    = "UpdateJob"
+	TerminateJob = "TerminateJob"
+	GetJob       = "GetJob"
 
 	// Job instance routes
-	CreateJobInstance = "CreateJobInstance"
-	DeleteJobInstance = "DeleteJobInstance"
-	GetJobInstance    = "GetJobInstance"
-	GetJobInstances   = "GetJobInstances"
-	GetJobPublicIPs   = "GetJobPublicIPs"
+	CreateJobInstance   = "CreateJobInstance"
+	DeleteJobInstance   = "DeleteJobInstance"
+	GetJobInstance      = "GetJobInstance"
+	GetJobInstances     = "GetJobInstances"
```

Comment:
I don't think this is used anymore. You renamed it to GetInstancesByJobID with I think is a better name.